runtime/compiler/control/JitDump.cpp

```diff
@@ -483,9 +483,86 @@ runJitdump(char *label, J9RASdumpContext *context, J9RASdumpAgent *agent)
 
    try
       {
-      // if our compinfo is null, we are an application thread
+      // Process user defined options first. The user can specify any method signature to be recompiled at an arbitrary
+      // dump event. This is typically used for debugging purposes so we process this option first if it exists.
+      if (NULL != agent->dumpOptions)
+         {
+         char *indexOfDot = strchr(agent->dumpOptions, '.');
+         char *indexOfParen = strchr(agent->dumpOptions, '(');
+
+         if ((NULL != indexOfDot) && (NULL != indexOfParen))
+            {
+            TR::VMAccessCriticalSection findUserMethod(TR_J9VMBase::get(jitConfig, crashedThread));
+
+            char *className = agent->dumpOptions;
```

Comment:
This can be `const char *className = ...`.

Reply:
Fixed in a36864f